odxtools/inputparam.py

```diff
@@ -40,12 +42,17 @@ def _build_odxlinks(self) -> Dict[OdxLinkId, Any]:
         return {}
 
     def _resolve_odxlinks(self, odxlinks: OdxLinkDatabase) -> None:
-        self._dop = odxlinks.resolve(self.dop_base_ref, DopBase)
+        self._dop_base = odxlinks.resolve(self.dop_base_ref, DopBase)
 
     def _resolve_snrefs(self, diag_layer: "DiagLayer") -> None:
         pass
 
     @property
-    def dop(self) -> DopBase:
+    def dop_base(self) -> DopBase:
```

Comment:
because the XML specifies a `DOP-BASE-REF`, not a `DOP-REF`. (I found this using my small reference checker script which ensures that every `.foo-ref` attribute comes with a corresponding `.foo`. amazingly, except for this location, everything else seemed to be okay...)